src/App/PropertyUnits.cpp

```diff
@@ -357,6 +357,79 @@ PropertyStiffness::PropertyStiffness()
     setUnit(Base::Unit::Stiffness);
 }
 
+//**************************************************************************
+//**************************************************************************
+// PropertyMass
+//++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+TYPESYSTEM_SOURCE(App::PropertyMass, App::PropertyQuantity)
+
+PropertyMass::PropertyMass()
+{
+    setUnit(Base::Unit::Mass);
+}
+
+//**************************************************************************
+//**************************************************************************
+// PropertyAngleOfFriction

```

Comment:
Why is the existing PropertyAngle not usable? I think we should use the existing properties.

Reply:
Sure, PropertyAngle is usable, but I don't know how to associate both Angle and AngleOfFriction units with a single property (and I don't see that it would make sense). Probably we should just not define a property for AngleOfFriction, so I have removed it.

Reply:
Then you misunderstood me. What you did is per se not wrong. AngleOfFriction exists already but we should have a property just called "angle" that can be used by all WBs and all kinds of angles.

Let's do it so: keep your PR as it is, I will merge and add a generic angle in a further PR. OK?

Reply:
OK, I will leave as is. There is a property already called Angle, I thought that is what you meant.